### DISCUSSION TITLE
Frequency selecting

### DISCUSSION THREAD

**Discussion by dr4g0nized** [egzumer/uv-k5-firmware-custom#281] [Ideas]
Hello! I would like to see version of this firmware with frequency selecting as on the last version of Fagci. I used default method and then tried the Fagci one, now i just can't use normal one. I know that you won't do these changes only for me, but i will be very pleased if you did it. Thanks

**Comment by ErikS-web** [comment]
Better read the manual and understand the way its implemented in EGZUMER-firmware.

What makes that other firmware-functionality so special that it would be an encore for this firmware

Or enjoy the vision of other firmware-builders, It's all hobby ;-)

**Reply by dr4g0nized** [reply]
Oh, i'm sorry. By the way, thank you for the reply. Have a nice day!

**Comment by ErikS-web** [comment]
Just be more specific in what your idea is and how you see it as an improvement.

Maybe you mean this. See #126
